app/src/main/java/com/example/ecodigify/dataclass/Recipe.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.ecodigify.dataclass
+
+data class Recipe(
+    val id: Int,
+    val name: String,
+    val thumb: String,
```

Comment:
```suggestion
    val thumbnail: Uri,
```

Reply:
fixed in https://github.com/Friedchicken-42/progettoIngegneriaSoftware/pull/3/commits/64326b6dd375162cdf0190fcbc17c6e6593ff96e